pygount/analysis.py

```diff
@@ -490,11 +496,27 @@ def __init__(
         folders_to_skip=pygount.common.regexes_from(DEFAULT_FOLDER_PATTERNS_TO_SKIP_TEXT),
         name_to_skip=pygount.common.regexes_from(DEFAULT_NAME_PATTERNS_TO_SKIP_TEXT),
     ):
-        self._source_patterns = source_patterns
+        self._is_git_link = False
+        self._source_patterns = self._set_temp_path_from_git_link(source_patterns)
         self._suffixes = pygount.common.regexes_from(suffixes)
         self._folder_regexps_to_skip = folders_to_skip
         self._name_regexps_to_skip = name_to_skip
 
+    def close(self):
+        # Remove temp dir if exists.
+        if self.is_git_link and os.path.isdir(self.source_patterns[0]):
+            try:
+                rmtree(self.source_patterns[0])
+            except OSError as e:
```

Comment:
I know it's anal but please rename `e` to `error`. It's just so much more expressive.

Reply:
Fixed in [86cbce5](https://github.com/roskakori/pygount/pull/111/commits/86cbce5e2f49caf80242b97c3aa5c0708b4abf03)